src/databricks/labs/remorph/reconcile/constants.py

```diff
@@ -76,3 +76,9 @@ class Constants:
             "target": HashAlgorithm.DATABRICKS_SHA_256.value,
         },
     }
+
+    jdbc_session_init = {
+        "oracle": """BEGIN dbms_session.set_nls('nls_date_format', '''YYYY-MM-DD''');
```

Comment:
should we have this in constants? I'm thinking it is better to build these inside specific source adapters only.

Reply:
Yes Sundar we can have separate constants for each source or in the specific source adapter.Need some input here.